### PR TITLE
Warehouse2

### PR DESCRIPTION
**Pull request by TumiMasilo**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -38,6 +38,7 @@ dependencies {
 
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
 
+<<<<<<< Updated upstream
```
Comment (TumiMasilo) [inline app/build.gradle]: remove

File: app/build.gradle [hunk 2]
```diff
@@ -59,5 +60,7 @@ dependencies {
 
 
 
+=======
```
Comment (TumiMasilo) [inline app/build.gradle]: remove

File: settings.gradle [hunk 3]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< Updated upstream
```
Comment (TumiMasilo) [inline settings.gradle]: remove

File: settings.gradle [hunk 4]
```diff
@@ -1,2 +1,6 @@
...
 include ':app', ':mssql-jdbc-7.4.1.jre8', ':SQL Driver', ':jtds-1.2.7'
+=======
+include ':app'
+>>>>>>> Stashed changes
```
Comment (TumiMasilo) [inline settings.gradle]: remove

**Review by TumiMasilo — COMMENTED** [review]
removed files

**Review by ThapeloNWU — CHANGES_REQUESTED** [review]
Please change your files

**Review by Patrick293 — CHANGES_REQUESTED** [review]
Yetsa Moo

**Review by ThapeloNWU — APPROVED** [review]

**Review by Patrick293 — APPROVED** [review]

**Pull request merged**